docs/api-docs/partner/subscriptions/foundation-guide.md

```diff
@@ -11,40 +11,45 @@ Additionally, it utilizes the [Channels toolkit](/api-docs/channels/guide/overvi
-> * Because we have a [store email address constraint](/api-docs/apps/guide/developer-portal#store-email-address-constraint?source=subscription-foundation) on draft and private apps, make sure that both your sandbox store and your Dev Portal account use the same email address. 
-> * The sandbox store must support multi-storefront sales.
+To configure your accounts you need to make sure the following requirements are met:
 
-1. To develop and test apps, you need a BigCommerce sandbox store. If you don't have one, [Create a Sandbox Store](/api-docs/partner/getting-started/create-a-sandbox-store?source=subscription-foundation).
-2. To register apps and create app API accounts, you need a BigCommerce Developer Portal account. If you don't have one, create a [Dev Portal account](https://devtools.bigcommerce.com/?source=subscription-foundation).
+1. Because there is a [store email address constraint](/api-docs/apps/guide/developer-portal#store-email-address-constraint?source=subscription-foundation) on draft and private apps, you need to make sure that both your sandbox store and your Dev Portal account use the same email address. 
+2. The sandbox store must support multi-storefront sales.
+3. To develop and test apps, you need a BigCommerce sandbox store. If you don't have one, [Create a Sandbox Store](/api-docs/partner/getting-started/create-a-sandbox-store?source=subscription-foundation).
+4. To register apps and create app API accounts, you need a BigCommerce Developer Portal account. If you don't have one, create a [Dev Portal account](https://devtools.bigcommerce.com/?source=subscription-foundation).
 
 ## Fork and install the source repository 
 
+As a first step you need to fork the source repository, clone it to your development, and install the dpackages for your app
+
+**To fork and install the source repository**
+
 1. Fork the [subscription-foundation repository (GitHub)](https://github.com/bigcommerce/subscription-foundation) to your GitHub account.
 2. Clone your fork to your local development environment.
-3. Navigate to your clone's root directory and install the application's default packages with npm.
+3. Navigate to your root directory of your cloned repository and install the default packages for your app by issuing the following command:
 
 ```shell title="Install packages"
 npm install
 ```
 ## Create an HTTPS tunnel
 
-You need to serve your app over fully-qualified publicly accessible URL to connect the app to your sandbox store. We recommend using [ngrok](https://ngrok.com/docs), a free tool that uses HTTP tunneling to securely expose `localhost` ports to the public internet. 
+After you have successfully installed the packages for your app, need to serve your app over a fully-qualified publicly accessible URL to connect the app to your sandbox store. We recommend using [ngrok](https://ngrok.com/docs), a free tool that uses HTTP tunneling to securely expose `localhost` ports to the public internet. 
```

Comment:
```suggestion
After you successfully install the packages, prepare to connect the app to your sandbox store. Because you must serve apps over fully-qualified publicly accessible URLs, this guide uses a tool called [ngrok](https://ngrok.com/docs) to open HTTP tunnels that securely expose `localhost` ports to the internet. 
```

The URLs will be plural because apps must expose multiple endpoints, `auth` and `load` at a minimum.

We don't have to solve this today, but this `serving apps over` pattern is in a lot of places and I wish we had a better way with a different preposition, or a different subject.

Reply:
also, past tense